packages/components/src/VirtualizedList/Row/Row.component.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import random from 'lodash/random';
+import {
+	defaultTableRowRenderer as DefaultTableRowRenderer,
+} from 'react-virtualized';
+import RowSelectionRenderer from '../RowSelection';
+import theme from './RowPlaceholder.scss';
+
+/**
+ * Higher order row-renderer that wrap the provided row renderer.
+ * It manages row selection classname and inject it to the row renderer props.
+ */
+function getRowRenderer({ selectionToggle, isSelected }) {
+	const RowTableRender = selectionToggle ?
+		RowSelectionRenderer( // eslint-disable-line new-cap
+			DefaultTableRowRenderer,
+			{
+				isSelected,
+				getRowData: rowProps => rowProps.rowData,
+			}) :
+		DefaultTableRowRenderer;
+
+	function Row(props) {
+		const { isScrolling } = props;
+
+		if (isScrolling) {
```

Comment:
Could you please use `props.isScrolling` instead of spreading the props?
That allocation is useless.